ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-isis-new.cfg

```diff
@@ -78,15 +78,22 @@ router isis EVPN_UNDERLAY
    mpls ldp sync default
    timers local-convergence-delay 15000 protected-prefixes
    advertise passive-only
+   spf-interval 250 10
+   graceful-restart
+   graceful-restart restart-hold-time 10
+   graceful-restart t2 level-1 10
+   graceful-restart t2 level-2 20
```

Comment:
Ordering is different on my device (cEOS 4.31)

```
router isis BLAH
   spf-interval 250 10
   graceful-restart
   graceful-restart t2 level-1 10
   graceful-restart t2 level-2 20
   graceful-restart restart-hold-time 10
```

Reply:
Fixed!